src/legacy/List/Filters.component.js

```diff
@@ -213,22 +216,34 @@ export default class Filters extends React.Component {
                     onInfoAction={isFiltering ? this.clearFilters : undefined}
                 >
                     <div style={{ padding: 10, margin: 10 }}>
-                        <div className="control-row checkboxes">
-                            <Checkbox
-                                className="control-checkbox"
-                                label={this.getTranslation("display_only_users_can_manage")}
-                                onCheck={this.setFilter("showOnlyManagedUsers", this.checkboxHandler)}
-                                checked={showOnlyManagedUsers}
-                            />
-
-                            <Checkbox
-                                className="control-checkbox"
-                                label={this.getTranslation("display_only_enabled_users")}
-                                onCheck={this.setFilter("showOnlyActiveUsers", this.checkboxHandler)}
-                                checked={showOnlyActiveUsers}
-                            />
-                        </div>
-
+                        <Grid container spacing={2} className="control-row">
+                            <Grid item xs={6} className="control-row checkboxes">
+                                <Checkbox
+                                    className="control-checkbox"
+                                    label={this.getTranslation("display_only_users_can_manage")}
+                                    onCheck={this.setFilter("showOnlyManagedUsers", this.checkboxHandler)}
+                                    checked={showOnlyManagedUsers}
+                                />
+                                <Checkbox
+                                    className="control-checkbox"
+                                    label={this.getTranslation("display_only_enabled_users")}
+                                    onCheck={this.setFilter("showOnlyActiveUsers", this.checkboxHandler)}
+                                    checked={showOnlyActiveUsers}
+                                />
+                            </Grid>
+                            <Grid item xs={6} className="control-row switch">
```

Comment:
I think this is a nice solution! For aesthetic reasons, how about having the Grid items with `xs={8}` and `xs={4}` respectively? (To have the switch button more to the right.)